aibolit/patterns/method_siblings/method_siblings.py

```diff
@@ -19,26 +19,28 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-from typing import List
-from aibolit.types_decl import LineNumber
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
-
-import javalang
+from aibolit.ast_framework.ast_node import ASTNode
+from typing import List
 import re
 
 
 class MethodSiblings:
-    def __init__(self):
-        pass
+    '''
+    Count those methods, which
+    start with the same names
+    '''
+    def _is_method_names_close(self, node: ASTNode, new_node: ASTNode) -> bool:
+        node_name = re.split('([A-Z][^A-Z]*)', node.name)
```

Comment:
It is better use `[0]` right here, it will allow to give better name to `node_name`. Maybe something like this
```suggestion
        common_prefix = re.split('([A-Z][^A-Z]*)', node.name)[0]
```
And change the next line accordingly.

Reply:
unfortunately i cant change it, because, after that I check `len(node_name) > 1`

Reply:
Then we better pick up a different name.